### PR TITLE
convert OfficeNameText.jsx to camel case #1875

### PR DESCRIPTION
**Pull request by fulin426**
### What github.com/wevote/WebApp/issues does this fix?
#1875 
Fix eslint warnings for WebApp/src/js/components/Widgets/OfficeNameText.jsx

### Changes included this pull request?
converted to camel case for OfficeNameText.jsx
Updated props for the following files:
src/js/components/Ballot/CandidateItem.jsx
src/js/components/VoterGuide/OrganizationPositionItem.jsx
src/js/components/VoterGuide/OrganizationVoterGuideCandidateItem.jsx
src/js/components/VoterGuide/VoterPositionItem.jsx

I was able to test in the browser CandidateItem and OrganizationPositionItem. I was not able to test OrganizationVoterGuideCandidateItem.jsx and VoterPositionItem.jsx because I couldn't find them in the browser. I updated it the same way so I think it should be okay.

### CODE REVIEW

**Comment by DaleMcGrew** [comment]
Hello @fulin426, I am doing a code review now. Did you get a chance to run tests on the code you changed within your local development environment?

File: src/js/components/Widgets/OfficeNameText.jsx [hunk 1]
```diff
@@ -24,34 +24,34 @@ export default class OfficeNameText extends Component {
...
-    if (political_party === undefined) {
+    const { contestOfficeName, politicalParty } = this.props;
+    if (politicalParty === undefined) {
       nameText = (
         <span className="no-political-party">
           <span>Candidate for </span>
-          { this.props.office_link ? (
-            <Link to={this.props.office_link}>
-              <span className="candidate-card-main__office">{ contest_office_name }</span>
+          { this.props.officeLink ? (
```
Comment (DaleMcGrew) [inline src/js/components/Widgets/OfficeNameText.jsx]: Is there any reason not to include `officeLink` in the deconstruction you do above on line 27?

Comment (fulin426) [reply src/js/components/Widgets/OfficeNameText.jsx]: No particular reason to not include offLink in the deconstruction. It wasn't included previously so I wasn't sure to include it in the refactor.

Comment (DaleMcGrew) [reply src/js/components/Widgets/OfficeNameText.jsx]: Could you please include `officeLink` in the deconstruction and update your pull request? Thank you!
Now I see your note saying you were able to do some testing. Sorry I missed that the first time I looked at this pull request.

Comment (fulin426) [reply src/js/components/Widgets/OfficeNameText.jsx]: I'll resubmit to include offLink in the deconstruction. 

I was able to test a few items in the browser's react extension to see the if the props were being based down correctly. Also, visually see if the party name, office, and link are being rendered. 

I am having some trouble finding OrganizationVoterGuideCandidateItem.jsx and VoterPositionItem.jsx. Any insight on where they are in the app?

Comment (DaleMcGrew) [reply src/js/components/Widgets/OfficeNameText.jsx]: WebApp/src/js/components/VoterGuide/OrganizationVoterGuideCandidateItem.jsx

...gets called by:
WebApp/src/js/components/VoterGuide/OrganizationVoterGuideCandidateList.jsx
(which is called by: WebApp/src/js/routes/VoterGuide/OrganizationVoterGuideOffice.jsx)
and:
WebApp/src/js/routes/VoterGuide/OrganizationVoterGuideCandidate.jsx

You can look up the paths in WebApp in this files:
WebApp/src/js/Root.jsx

Examples are:
OrganizationVoterGuideCandidate: http://localhost:3000/candidate/wv02cand40137/btvg/wv01org14

OrganizationVoterGuideOffice: http://localhost:3000/office/wv02off19951/btvg/wv01org14

I can show you if you would like to join the hangout (you can ask me written questions via Slack, and I can respond verbally.)

Comment (fulin426) [reply src/js/components/Widgets/OfficeNameText.jsx]: Thanks for showing me where the paths are. It'll help in the future on finding where things are. I'll send more messages through Slack next time. 

I also included officeLink in the destructuring  and things looked okay on my end.

**Comment by DaleMcGrew** [comment]
Thank you @fulin426! 👍

**Pull request merged**